### PR TITLE
State Data model update

### PR DESCRIPTION
**Pull request by acriptis**
Add get_or_create method for the User models;
Create SkillResponse data model

### CODE REVIEW

File: deeppavlov/core/agent_v2/state_schema.py [hunk 1]
```diff
@@ -13,6 +14,33 @@ def to_dict(self):
...
+            *args:
+            kwargs:
+
+        Returns: tuple (instance:User, is_created:bool)
+            or raises Exception if Multiple instances are found
+        """
+        results = cls.objects(*args, **kwargs)
+        if results:
+            if len(results) > 1:
+                # raise Exception
```
Comment (my-master) [inline deeppavlov/core/agent_v2/state_schema.py]: Could you please remove the comments? It seems like they were made for debugging purposes.

File: deeppavlov/core/agent_v2/state_schema.py [hunk 2]
```diff
@@ -13,6 +14,33 @@ def to_dict(self):
...
+    def get_or_create(cls, *args, **kwargs):
+        """
+        gets or creates an object from init specification
+
+        Args:
+            *args:
+            kwargs:
+
+        Returns: tuple (instance:User, is_created:bool)
+            or raises Exception if Multiple instances are found
```
Comment (my-master) [inline deeppavlov/core/agent_v2/state_schema.py]: Google style docstrings is accepted in the project. Please make yourself familiar with it:
https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html
There is a special syntax for raised Exceptions in docstrings.

File: deeppavlov/core/agent_v2/state_schema.py [hunk 3]
```diff
@@ -73,3 +101,19 @@ def to_dict(self):
...
         }
+
+
+class SkillResponse(DynamicDocument):
+    """
+    Data Model for Skills responses. It specifies behavior hypothesis from particular skill
+    after processing new Utternace
+    """
+    text = StringField()
+    confidence = FloatField()
```
Comment (my-master) [inline deeppavlov/core/agent_v2/state_schema.py]: I think, we could set default value to 1.0 here.

File: deeppavlov/core/agent_v2/state_schema.py (hunk 1 above)
Comment (acriptis) [reply deeppavlov/core/agent_v2/state_schema.py]: fixed

File: deeppavlov/core/agent_v2/state_schema.py (hunk 2 above)
Comment (acriptis) [reply deeppavlov/core/agent_v2/state_schema.py]: fixed

File: deeppavlov/core/agent_v2/state_schema.py (hunk 3 above)
Comment (acriptis) [reply deeppavlov/core/agent_v2/state_schema.py]: fixed

**Pull request closed without merging**